content/blog/2017/08/2017-08-17-speaker-blog-blazemeter.adoc

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: "Running load tests in Jenkins Pipeline with Taurus"
+tags:
+- event
+- jenkinsworld
+author: hinman
+---
+
+[NOTE]
+====
+This is a guest post by Guy Salton, Sr. Professional Services Engineer for
+link:https://www.blazemeter.com/ca-technologies[CA BlazeMeter].
+====
+
+link:https://www.blazemeter.com/blog/running-jmeter-test-jenkins-pipeline-tutorial[Jenkins
+Pipeline] is an important Jenkins feature for creating and managing your
+Jenkins project as a
+link:https://www.blazemeter.com/blog/groovy-new-black[Groovy] script. This is
```

Comment:
Not sure it's a good idea calling the Pipeline DSL "Groovy" given its limitations.